docs/tutorial/workers.md

```diff
@@ -214,24 +130,37 @@ Prefect will build a custom Docker image containing your workflow code that the
         get_repo_info.deploy(
             name="my-first-deployment", 
             work_pool_name="my-docker-pool", 
-            image=DeploymentImage(
-                name="my-first-deployment-image",
-                tag="tutorial",
-                dockerfile="Dockerfile"
-            ),
+            image="my-first-deployment-image:tutorial",
             push=False
         )
     ```
 
+    For this tutorial, your Docker worker is running on your machine, so you don't need to push the image built by `flow.deploy` to a registry. When your worker is running on a remote machine, you need to [push the image to a registry that the worker can access](/guides/prefect-deploy/#creating-work-pool-based-deployments-with-deploy).
+
+1. Deploy your flow to the work pool:
+
+    <div class="terminal">
+
+    ```bash
+    python repo_info.py
+    ```
+
+    </div>
+
+Prefect builds a custom Docker image containing your workflow code that the worker can use to dynamically spawn Docker containers whenever this workflow needs to run.
+
 ### Modify the deployment
 
-If you need to make updates to your deployment, you can do so by modifying your script and rerunning it. You'll need to make one update to specify a value for `job_variables` to ensure your Docker worker can successfully execute scheduled runs for this flow. See the example below.
+To modify your deployment, edit your script and run it again. 
+
+## Customize infrastructure settings
 
-The `job_variables` section allows you to fine-tune the infrastructure settings for a specific deployment. These values override default values in the specified work pool's [base job template](/concepts/work-pools/#base-job-template).
+Customize infrastructure settings for specific deployments by updating the `job_variables` section:
```

Comment:
The important point here is that you're overriding the work pools' defaults, so I'd leave that part in.